spec/chewy/search/scrolling_spec.rb

```diff
@@ -33,6 +33,56 @@
     let(:countries) { Array.new(3) { |i| Country.create!(rating: i + 2, name: "country #{i}") } }
 
     describe '#scroll_batches' do
+      describe 'with search backend returning failures' do
```

Comment:
btw, do you think it will be possible to provide integration spec? Where you really call ES and get real answer?

Reply:
Do you have a suggestion how I can set up an integration spec that returns this error state? Elasticsearch should return a total count higher than zero but then return zero hits in order to hit this code path.